Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -969,7 +969,7 @@ def genS(fragment,howMuch):
         {"cfg":"SingleNuE10_cfi",
          "--fast":"",
          "--conditions":"auto:run2_mc",
-         "-s":"GEN,SIM,RECOBEFMIX,DIGI,L1,DIGI2RAW",
+         "-s":"GEN,SIM,RECOBEFMIX,DIGI",
          "--eventcontent":"PREMIX",
          "--datatier":"GEN-SIM-DIGI-RAW",
```

Comment:
Hi @makortel 
should datatier be changed too?

Reply:
Probably, thanks for catching it. Does anyone have a good suggestion? The event content is more or less "DIGI", so should the data tier be `DIGI` as well?

Reply:
What does it mean exactly your "more or less DIGI"? :-) 

if you are proposing  GEN-SIM-DIGI format then yes, you should change it.

 the pmx library format for 2018 relvals has been 
.._PREMIXUP15_PU25/.../GEN-SIM-DIGI-RAW
so far and so input the format for FastSim and FullSim PU with premix.  Did you consider it?

Reply:
> What does it mean exactly your "more or less DIGI"?

I mean that the events contain DIGI collections plus some of whose classification I'm not sure (like `genPUProtons` and TrackingParticles etc).

> if you are proposing GEN-SIM-DIGI format then yes, you should change it.

No, I meant only "DIGI". The events do not contain any GEN or SIM information (unless the aforementioned `genPUProtons` and TrackingParticles are classified as such).

Reply:
Thanks for reminding of the input side as well. My remaining question is then should the data tier be changed or not, and if yes, to what? (I'm not sure though who should answer)

Reply:
I see I've defined it as `PREMIX`
https://github.com/cms-sw/cmssw/blob/6c981941f8c6367e308bb58ff4eccebe009d3670/Configuration/PyReleaseValidation/python/relval_steps.py#L2785-L2789
but that is really more a random string rather than a deliberate choice. I think the issue should be decided here and then propagated to all premixing cases.

(note that so far the phase2 premixing has been exercised only in IBs and PR tests, not even in RelVals)

Reply:
hhmmm interesting.. so.. I suggest a "real life" test in relval to check the technicalities of your change.

Reply:
@fabiocos @davidlange6 Could you please advise on the data tier question? (i.e. what data tier should be used for the premixed pileup library after this PR)

Reply:
The only official documentation for data tiers that I can find is: https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookDataFormats#AboutTiers

Since the premixing output is (at least somewhat) different from the regular DIGI output, I think it makes sense to define a PREMIX datatier and use that everywhere.

Reply:
I'm fine with a new `PREMIX` data tier as well. I suppose that would be a bit more future-proof in a sense that if we add some GEN/SIM products (like switching muon detectors to use SimHits) we don't have to think about the proper data tier again.

Any objections?

Reply:
No comments within a week so I changed the premixing library data tier to `PREMIX`.